### DISCUSSION TITLE
Issue of location not on campus

### DISCUSSION THREAD

**Discussion by dreamchaser123** [community/community#56735] [GitHub Education]
### Hi everyone,

Hiiiii guys, I'm a student of hkust and hkust(gz), and now I'm taking summer vacation, I won't go to school until Sep, but I really wanna use copilot, is there anyone can help me verify my student status? Thank you so much!

**Comment by samus-aran** [comment]
Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
